packages/composite-checkout/src/lib/line-items.js

```diff
@@ -25,3 +25,8 @@ export function useTotal() {
 	const [ , total ] = useLineItems();
 	return total;
 }
+
+export function useTax() {
```

Comment:
Good idea! I wonder, though, if this should just be a generalized hook like `useLineItemsOfType('tax')` and/or `useFirstLineItemOfType('tax')` (there may be better names) since `composite-checkout` has no concept of "tax" types (or any type at all, really). Types are currently designed to be for the use of the consumer.